_maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm

```diff
@@ -3367,21 +3367,24 @@
 /area/ruin/unpowered/syndicate_lava_base/engineering)
 "jv" = (
 /obj/effect/turf_decal/bot,
-/obj/structure/closet/toolcloset,
 /obj/effect/decal/cleanable/dirt,
+/obj/machinery/suit_storage_unit/radsuit{
+	name = "radiation suit storage unit"
```

Comment:
Why not have this defined by the object?

Reply:
Good idea